service_configuration_lib/spark_config.py

```diff
@@ -1036,14 +956,19 @@ def _convert_user_spark_opts_value_to_str(user_spark_opts: Mapping[str, Any]) ->
     return output
 
 
+def update_spark_srv_configs(spark_conf):
```

Comment:
Nit: perhaps we can add type hints for newly added functions? `spark_conf: Mapping[str, str]`